common/ellipses.js

```diff
@@ -293,13 +301,13 @@
 
                         // This function checks for height of an element in the row at an index(td'th)
                         // and Set overflow
-                        var updateHeight = function(index ,element) {
+                        var updateHeight = function(index, element) {
                             var height = element.clientHeight;
                             if (height > maxHeight) {
                                 scope.overflow[index] = true;
                                 scope.hideContent = true;
                                 containsOverflow = true;
-                                scope.maxHeightStyle =  maxHeightStyle;
+                                scope.maxHeightStyle = maxHeightStyle;
                             } else {
                                 scope.overflow[index] = false;
                             }
```

Comment:
[resizeRow](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/ellipses.js#L320) should be wrapped.